src/linter.js

```diff
@@ -1,50 +1,41 @@
 const { Runner, run } = require('tslint/lib/runner');
-const path = require('path');
-const chalk = require('chalk');
 
 const options = JSON.parse(process.argv[2]) || {};
 
-function log(message) {
-  if (!options.silent) {
```

Comment:
We'll want to keep this option in case people want to suppress all output.

Reply:
I know, I had it on my todo list to handle it in the plugin.js and have forgotten to add it. Will add it.